timemachine/fe/topology.py

```diff
@@ -476,13 +458,13 @@ def setup_end_state(self):
         )
         bond_potential = mol_hb.bind(mol_bond_params)
         angle_potential = mol_ha.bind(mol_angle_params)
-
-        torsion_params = np.concatenate([mol_proper_params, mol_improper_params])
-        torsion_idxs = np.concatenate([mol_pt.idxs, mol_it.idxs])
-        torsion_potential = potentials.PeriodicTorsion(torsion_idxs).bind(torsion_params)
+        proper_potential = mol_pt.bind(mol_proper_params)
+        improper_potential = mol_it.bind(mol_improper_params)
         nonbonded_potential = mol_nbpl.bind(mol_nbpl_params)
 
-        system = VacuumSystem(bond_potential, angle_potential, torsion_potential, nonbonded_potential, None, None)
+        system = VacuumSystem(
```

Comment:
@mcwitt - any idea why `mypy` permits this to be to `None`, `None` (since we removed the `Optional` qualifier)

Reply:
I think this is because we've configured mypy to ignore skip checking untyped functions/methods: https://github.com/proteneer/timemachine/blob/1e1ce2f6b663e1129b4e67471f005cea060e3573/pyproject.toml#L20

If you annotate `setup_end_state` as returning `VacuumSystem`, I think mypy will complain.

Reply:
Ah, thanks. Not sure how I feel here.. since this function sets up a non-chiral state to begin with, so `None` seems less technically correct but simpler.

Reply:
In the spirit of correctness, and since we're now "bare" in terms of downstream assertions, I'll be verbose.

Reply:
Added return types and removed the bare `None, None`.